README.md

```diff
@@ -1,30 +1,67 @@
 # Brian's Blueprints
+
 Brian's Blueprints for Factorio
 
-Don't just look at the latest release shown on the left.  Each book is released separately so to see them you have to go to the [all-releases page](https://github.com/bcwhite-code/brians-blueprints/releases).
+Don't just look at the latest release shown on the left. Each book is released separately so to see them you have to go to the [all-releases page](https://github.com/bcwhite-code/brians-blueprints/releases).
+
+Blueprints include:
+
+- [Brian's Bootstrap](./brians-bootstrap/)
+- [Brian's Trains](./brians-trains/)
+- [Brian's Trains Auxiliary](./brians-trains-auxiliary/)
+- [Finite State Machine](./finite-state-machine/)
+- [SBF Big Blocks](./sbf-big-blocks/)
+- [Self Building Angel Bob](./self-building-angelbob/)
+- [Self Building Factory](./self-building-factory/)
+- [Self Building Rocket](./self-building-rocket/)
+- [Self Building Space Exploration](./self-building-spacex/)
+- [Tileable Reactor](./tileable-reactor/)
 
 ## Commands
-The "fbp.py" script can be used to split books into a directory structure and later build them back into a single entity as well as to unpack/pack single blueprints from their encoded form into json.  The latter allows working with single blueprints within a book.
 
-#### Split a Book
-```
+The "fbp.py" script can be used to split books into a directory structure and later build them back into a single entity as well as to unpack/pack single blueprints from their encoded form into json. The latter allows working with single blueprints within a book.
```

Comment:
It's now only `Fatul`.  `fbp` is no longer used so reference to it can be removed.

Reply:
I was under the assumption that `Fatul` probably doesn't split BPs. I took a careful look & found that the command names are just different.

I will add `Fatul` equivalent commands & remove `fbp` commands

Reply:
Resolved in [db24606](https://github.com/bcwhite-code/brians-blueprints/pull/21/commits/db24606b5f0f5026e0f7d802ab4e6ad18610f508)